config.py

```diff
@@ -0,0 +1,34 @@
+from os.path import abspath, dirname, join
```

Comment:
nit: It would be very useful to have here (or somehere else) to link in the awesome doc that describes why the config is structured like this.